suite-common/toast-notifications/src/notificationsActions.ts

```diff
@@ -37,6 +43,15 @@ export const addToast = createActionWithExtraDeps(
     }),
 );
 
+// Adds a Toast if there is not one of same type visible.
```

Comment:
The comment is misleading, it adds the toast and just close all with the same type. I don't think it's the desired behaviour, check the video where I simulated `tor-is-slow` once per second:

https://user-images.githubusercontent.com/3729633/213138548-26d66412-1d03-41fb-a65b-c5cd5b709bfb.mov

Reply:
Ahh yes, the comment is from the first approach I took, it is wrong.
But then I thought that it is probably better to show a new popup when Tor is still being misbehaving. The idea was to have only one but to keep it there while it is misbehaving. 

But I am not really sure what the behavior should be.

Reply:
I have come into a situation that replicates the above video.

When a notification is open, do not display another.

Reply:
@matejkriz done https://github.com/trezor/trezor-suite/pull/7397/commits/8ac2d0638bfc2807405d3d3c5451c3dcd48a5893